tests/Sniffs/Complexity/data/cognitive/nestng.php

```diff
@@ -0,0 +1,15 @@
+<?php
```

Comment:
Don't understand the file name.

Reply:
filename typo -> renamed to "nesting.php"